velox/connectors/hive/TableHandle.cpp

```diff
@@ -145,6 +145,20 @@ std::string HiveTableHandle::toString() const {
   if (dataColumns_) {
     out << ", data columns: " << dataColumns_->toString();
   }
+  if (!tableParameters_.empty()) {
+    std::map<std::string, std::string> orderedTableParameters{
```

Comment:
tableParameters_ is an unordered map so s/orderedTableParameters/tableParameters/